edx_solutions_api_integration/users/views.py

```diff
@@ -353,6 +364,9 @@ def get(self, request, *args, **kwargs):
         """
         GET /api/users?ids=11,12,13.....&page=2
         """
+        course_id = self.request.query_params.get('include', None)
```

Comment:
Use a descriptive request param instead of `include`

Reply:
Addressed